File: 2303/LAB5/Student.cpp
//
// Created by Kush Shah on 2/19/20.
//

#include "Student.h"


Student::Student(string name, int ID) {
this->ID=ID;
this->name=name;
}

Student::~Student() {

}

string Student::getName() const {
    return name;
}

int Student::getId() const {
    return ID;
}

File: 2223 Algorithms/Homework/HW2/Q1/Student.java
public class Student implements Comparable<Student>
{

    public int id;

    public String name;

    public String year;

    public Student(int id, String name, String year)
    {
        this.id = id;
        this.name = name;
        this.year = year;
    }

    public int compareTo(Student std)
    {
        return this.id - std.id;
    }
}
